pkg/controller/framework/controller_test.go

```diff
@@ -371,11 +366,18 @@ func TestUpdate(t *testing.T) {
 		},
 	}
 
-	// run every test a few times, in parallel
 	const threads = 3
+	testDoneWG.Add(threads * len(tests))
```

Comment:
Sure. My intention was to keep `const threads = 3` close to the code that uses it below, but what you suggest would mean fewer line changes so it makes sense.